requirements-azure.txt

```diff
@@ -12,6 +12,7 @@ azure-mgmt-containerinstance==1.4.0
 azure-mgmt-containerregistry==2.0.0
 azure-mgmt-containerservice==9.1.0
 azure-mgmt-datalake-store==0.5.0
+azure-mgmt-diagnosticsetting==0.1.0
```

Comment:
There is no 'azure-mgmt-diagnosticsetting', this module depend on 'azure-mgmt-monitor', please delete!
```suggestion
```